### ISSUE TITLE
Documentation is not a tree but a forest

### ISSUE THREAD

**Issue by thigg** [sailfishos/sailfish-components-webview#158]
At least the [sailfish-webview-webviewadditional.qdoc](https://github.com/sailfishos/sailfish-components-webview/blob/master/doc/sailfish-webview-webviewadditional.qdoc) site does not seem to be linked anywhere else in the documentation. I just found it via some search voodoo with a duck. (I was searching for the documentation of `onRecvAsyncMessage` and `addMessageListener`)

Flypig hat in the [community news](https://forum.sailfishos.org/t/sailfish-community-news-18th-november-suomenlinna-webview) a very good piece that might be worth adding to the documentation.

**Comment by thigg** [comment]
Also add documentation about the [CORS flags ](https://forum.sailfishos.org/t/webview-webengine-cors-headers-and-file-loading-in-sfos-4-4)

**Comment by llewelld** [comment]
> At least the [sailfish-webview-webviewadditional.qdoc](https://github.com/sailfishos/sailfish-components-webview/blob/master/doc/sailfish-webview-webviewadditional.qdoc) site does not seem to be linked anywhere else in the documentation.

We did [very recently](https://github.com/sailfishos/docs.sailfishos.org/commit/2109b34a3bc35b28ed3d1ab724fa86e062f0b6bd) add a link from the [docs site](https://docs.sailfishos.org/Develop/Apps/#sailfish-webview-api) to the WebView documentation. Is that the kind of thing you were thinking of?

We're always looking for ways to improve the navigability of our documentation forest of course, so please do submit a PR to docs.sailfishos.org if you can see some nice way to make it easier to find things.

> Also add documentation about the [CORS flags ](https://forum.sailfishos.org/t/webview-webengine-cors-headers-and-file-loading-in-sfos-4-4)

Yes, a good point, we should add that here.

**Comment by poetaster** [comment]
Now if only I could find that in the repo!

**Comment by poetaster** [comment]
> Which repo/part do you mean exactly? If it's a page on docs.sailfishos.org and you have the page itself, the easiest way to find it in the repo is to hit the "Edit this page" link at the bottom of the page.

I found the readme.md in Apps but I failed to find the source of:  https://sailfishos.org/develop/docs/sailfish-components-webview/

**Comment by thigg** [comment]
yeah also noticed today that github search doesnt find stuff thats obviously there
